source/_posts/2017-09-29-hacktoberfest.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Participating in Hacktoberfest"
+description: "Home Assistant will be participating in Hacktoberfest to help people to get started with open-source."
+date: 2017-09-29 08:00:00 +0000
+date_formatted: "September 29, 2017"
+author: FAbian Affolter
+author_twitter: fabaff
+comments: true
+categories: Community
+og_image: /images/blog/2017-09-hacktoberfest/hacktoberfest-2017.png
+---
+
+Home Assistant will join this year for [Hacktoberfest], an event organised by DigitalOcean and GitHub to support and celebrate open source. The idea is that open source projects like Home Assistant will gather a bunch of entry-level bugs, features, and documentation enhancements and that you, a current or future contributor, will fix them. If you submit four pull-requests during the month of October you will have earned yourself a limited edition Hacktoberfest T-shirt!
```

Comment:
- `Home Assistant will join this year for [Hacktoberfest]` -> I am having trouble coming up with the correct grammar for this. Does anyone else have ideas?
- `will fix them` -> `will help us fix them`

Reply:
How about: `Home Assistant joins Hacktoberfest this year! Hacktoberfest is an event....`

Reply:
`Home Assistant will participate in [Hacktoberfest]`?